CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [Next version]
 
+## [5.2.0] - 2019-05-29
```

Comment:
needs to be updated (on master too I guess)

Reply:
done and the PR to update master here: https://github.com/onfido/onfido-sdk-ui/pull/674